docs/data/charts/axis/GroupedAxesTickSize.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+
+export default function GroupedAxesTickSize() {
```

Comment:
Should this be a playground with the ability to customize the tick sizes? It will probably make it clear how the tick size impacts the visual result.

Reply:
I tried it, but it looks cramped and not very useful, the user can edit the preview if they want though